src/components/Leaderboard/TableBody/styled.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { colors } from '../../../../ui/colors';
+
+export const BodyTr = styled.tr`
+    background: ${colors.primary};
+`;
+
+export const BodyTd = styled.td`
+    width: 33%;
+    height: 50px;
+    &:first-child {
```

Comment:
❗️
Это разные сущности, давай визуально разделим
```suggestion
    height: 50px;
    
    &:first-child {
```